CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 
-## [v3.3.0]\[FORKED\](https://github.com/ischemaview/phonegap-plugin-push/tree/v3.3.0) (2022-12-14)
+## [v3.4.0](https://github.com/ischemaview/phonegap-plugin-push/tree/v3.4.0) (2023-02-25)
```

Comment:
```suggestion
## [v3.4.1](https://github.com/ischemaview/phonegap-plugin-push/tree/v3.4.1) (TBD)
* Remove handling notification arrival when app is in foreground (no longer showing in-app banner)
* (iOS) Pipe notifications banner tapped on foreground to `notification` events on javascript side

## [v3.4.0](https://github.com/ischemaview/phonegap-plugin-push/tree/v3.4.0) (2023-02-25)
```

Reply:
@franco-octavianus I was going to open a separate PR with the tag and change log.